labs/building-container-images.md

```diff
@@ -0,0 +1,115 @@
+# Building the Docker Image
+
+In this lab, we'll use a build trigger to connect Google Source Repository to Google Container Registry.
+After this module, whenever we tag an image for release, we'll automatically kick off a spinnaker deployment.
+
+## Creating your Source Code Repository
+
+1. Download the source code:
+
+    ```source
+    wget https://gke-spinnaker.storage.googleapis.com/sample-app.tgz
+   ```
+
+1. Unpack the source code
+  
+    ```shell
+    tar xzfv sample-app.tgz
+    ```
+
+1. Go into the source code directory
+
+    ```shell
+    cd sample-app
+    ```
+
+1. Set the username and email address for your Git commits in this repository.
+
+    > **NOTE:** Replace [EMAIL_ADDRESS] with your Git email address.
+
+    ```shell
+    git config --global user.email "[EMAIL_ADDRESS]"
+    ```
+
+    > **NOTE:** Replace [USERNAME] with your Git username.
+
+    ```shell
+    git config --global user.name "[USERNAME]"
+    ```
+
+1. Make the initial commit to your source code repository
+
+    ```shell
+    git init
+    git add .
+    git commit -m "Initial commit"
+    ```
+
+1. Create a Cloud Source Repository to host your code
+
+    ```shell
+    gcloud source repos create sample-app
+    git config credential.helper gcloud.sh
+    ```
+
+1. Add your newly created repository as a remote:
+
+    ```shell
+    export PROJECT=$(gcloud info --format='value(config.project)')
+    git remote add origin https://source.developers.google.com/p/$PROJECT/r/sample-app
```

Comment:
If we pull from this repo we will need to call this google instead of origin:
```
git remote add google https://source.developers.google.com/p/$PROJECT/r/sample-app
```